src/hooks/screen/useChatScreenHook.ts

```diff
@@ -68,12 +92,18 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
           userId,
           selectedChannel?.id,
           message,
-          localDb
+          localDb,
+          replyData ? MESSAGE_TYPE_REPLY : MESSAGE_TYPE_REGULAR,
+          'pending',
+          json
         );
         currentChatSchema.save(localDb);
         refresh('chat');
         refresh('channelList');
       }
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendChat` function is updated to handle the new `reply_data` property. Ensure that this change does not introduce any inconsistencies or errors in the chat functionality.
```diff
+ currentChatSchema = await ChatSchema.create(
+           userId,
+           selectedChannel?.id,
+           message,
+           localDb,
+           replyData ? MESSAGE_TYPE_REPLY : MESSAGE_TYPE_REGULAR,
+           'pending',
+           json
+          );
+          currentChatSchema.save(localDb);
+          refresh('chat');
+          refresh('channelList');
+        }
+ 
+       clearReplyPreview();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->